### PR TITLE
Drop download option (mobile net is fast nowadays)

### PR DESCRIPTION
**Pull request by TomasVotruba**
I think we can drop this feature, since using ZIP is kinda useless today. Internet is just fine, everywhere.

### CODE REVIEW

File: src/Configuration/ConfigurationOptionsResolver.php [hunk 1]
```diff
@@ -51,7 +50,9 @@ class ConfigurationOptionsResolver
         CO::TREE => true,
         // helpers
         CO::VISIBILITY_LEVELS => [],
-        CO::SOURCE_CODE => ''
+        CO::SOURCE_CODE => '',
+        // removed, but BC for templates
+        'download' => false
```
Comment (jankal) [inline src/Configuration/ConfigurationOptionsResolver.php]: I came here to loose BC. So we loose BC.

File: tests/Console/Command/GenerateCommandExecuteTest.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ public function testExecute()
         $inputMock->shouldReceive('getOptions')->andReturn([
             'config' => null,
             'destination' => TEMP_DIR . '/Api',
-            'source' => __DIR__ . '/Source'
```
Comment (jankal) [inline tests/Console/Command/GenerateCommandExecuteTest.php]: What? Not related to PR.

Your IDE did do this. Didin't it?

File: tests/Console/Command/GenerateCommandExecuteTest.php [hunk 3]
```diff
@@ -63,7 +65,9 @@ public function testExecuteWithError()
 
         $this->assertSame(
             1, // failure
-            MethodInvoker::callMethodOnObject($this->generateCommand, 'execute', [$inputMock, $outputMock])
+            MethodInvoker::callMethodOnObject(
```
Comment (jankal) [inline tests/Console/Command/GenerateCommandExecuteTest.php]: This whole file has been fixed by your IDE. This is not the PR to do this.

**Comment by jankal** [comment]
@TomasVotruba could you please aggregate all drops of features into one branch and so into one PR?

File: tests/Console/Command/GenerateCommandExecuteTest.php (hunk 3 above)
Comment (TomasVotruba) [inline tests/Console/Command/GenerateCommandExecuteTest.php]: It was fixed by coding standard actually.

**Comment by TomasVotruba** [comment]
Thank you!

**Pull request merged**